.circleci/src/jobs/bump_version.yml

```diff
@@ -11,11 +11,11 @@ steps:
   - restore_yarn_tools_cache
   - run:
       name: Calculate next version name
-      command: echo "export NEW_VERSION_NAME=$(yarn --silent next-version calc | jq .versionName)" >> ${BASH_ENV}
+      command: echo "export NEW_VERSION_NAME=$(yarn next-version calc | jq .versionName)" >> ${BASH_ENV}
```

Comment:
Is it now `silent` as default?
Edit: Looks good anyway, seems like there is no such flag anymore.